kube_downscaler/resources/stackset.py

```diff
@@ -0,0 +1,26 @@
+import json
+
+from pykube.objects import NamespacedAPIObject
+
+from .scalable import Scalable, DOWNSCALER_SAVED_ANNOTATION
+
+
+class StackSet(NamespacedAPIObject, Scalable):
```

Comment:
Aha! A major new feature you sneaked into this PR :smirk:

Reply:
Please mention/refer the github repo of StackSetController in Python comments (other readers might be unaware what this is about).

Reply:
Updated the PR

Reply:
We need to check what happens if StackSet is not registered as CRD in the cluster --- it should not just fail...

Reply:
The support for stackset needs to be enabled explicitly by passing it to the `--kind` flag. Without it stacksets are not queried because the only resource kinds which are downscaled by default are deployments. Tested locally on a minikube cluster.